extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
+The Reader ID is constructed similarly to the mechanism used to build ExternalCID described [here](https://docs.google.com/document/d/1f7z3X2GM_ASb3ZCI_7tngglxwS6WoWi1EB3aKzdf6vo/edit#heading=h.hb9q0wpwwhuf). An example Reader ID is `amp-OFsqR4pPKynymPyMmplPNMvxSTsNQob3TnK-oE3nwVT0clORaZ1rkeEz8xej-vV6`.
 
-The JSON-LD and Microdata formats are supported.
+### Configuration
 
-## JSON-LD markup
+The `amp-subscriptions` extension must be configured using JSON configuration:
 
-Using JSON-LD, the markup would look like:
+<table>
+  <tr>
+    <th>Property</th>
+    <th>Values</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>services</code></td>
+    <td>&lt;array&gt; of &lt;object&gt;</td>
+    <td>This <code>array</code> must include:<ul><li>One <a href="#local-service">Local Service</a></li><li>Zero or more <a href="#vendor-services">Vendor Services</a>.</li></ul></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>score</code></td>
+    <td>&lt;object&gt;</td>
+    <td>Determines which service is selected if no valid entitlements are returned.<br/>See <a href="#service-score-factors">Service Score Factors</a> for more details.</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><code>fallbackEntitlement</code></td>
+    <td>&lt;object&gt;</td>
+    <td>Determines what level of access the Reader should have if all services fail to respond to the Authorization requests.<br/>See <a href="#fallback-entitlement">Fallback Entitlement</a> for more details.</td>
+  </tr>
+</table>
+
+Here is an example of a configuration:
 
 ```html
-<script type="application/ld+json">
+<script type="application/json" id="amp-subscriptions">
   {
-    "@context": "http://schema.org",
-    "@type": "NewsArticle",
-    "isAccessibleForFree": false,
-    "publisher": {
-      "@type": "Organization",
-      "name": "The Norcal Tribune"
+    "services": [
+      {
+        // Local service (required)
+        "authorizationUrl": "https://pub.com/amp-authorisation?rid=READER_ID&url=SOURCE_URL",
+        "pingbackUrl": "https://pub.com/amp-pingback?rid=READER_ID&url=SOURCE_URL",
+        "actions":{
+          "login": "https://pub.com/amp-login?rid=READER_ID&url=SOURCE_URL",
+          "subscribe": "https://pub.com/amp-subscribe?rid=READER_ID&url=SOURCE_URL"
+        }
+      },
+      {
+        // Vendor services (optional)
+        "serviceId": "service.vendor.com"
+      }
+    ],
+    "score": {
+      "supportsViewer": 10,
+      "isReadyToPay": 9
     },
-    "hasPart": {...},
-    "isPartOf": {
-      "@type": ["CreativeWork", "Product"],
-      "name" : "The Norcal Tribune",
-      "productID": "norcal_tribune.com:basic"
+    "fallbackEntitlement": {
+      "source": "fallback",
+      "granted": true,
+      "grantReason": "SUBSCRIBER",
+      "data": {
+        "isLoggedIn": false
+      }
     }
   }
 </script>
 ```
 
-Thus, notice that:
+### Local service
+
+The following properties are defined in this configuration:
```

Comment:
What configuration? The same one as above?

Reply:
What is a local service vs a local configuration? If the configuration example below is for a local service what is the above configuration for?

Reply:
This is diving into the "local service" part of the overall configuration.  I will add some explanatory text at the beginning of the section.

Reply:
Changed to:
>The local service is provided by the Publisher to control and monitor access to documents.
>
>It is configured using the following properties: